### PR TITLE
RELATED: ONE-3791 Documentation for 7.0.0

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/76/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/57/ : SUCCESS in 9s

File: docs/migration_guide.md [hunk 1]
```diff
@@ -1,110 +1,10 @@
...
-        colors: ['rgb(195, 49, 73)', 'rgb(168, 194, 86)']
-    }}
-/>
-```
-
-## React 16 support
-We updated Gooddata.UI to React 16. We advise you to update your application as well to avoid package duplicity and potential issues. Update also all your packages with React dependency, such as "react-dom", "react-router", etc.
+## Typescript upgraded
+Typescript version 3.3.4000 is used for building.
+If you are using this library from typescript project, it is necessary to upgrade to typescript of version at least 3.3.4000.
```
Comment (evgeniabogranskaya) [inline docs/migration_guide.md]: If you are using this library from the typescript project, upgrade it to version 3.3.4000 at least.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/77/ : SUCCESS in 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/58/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/83/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/64/ : SUCCESS in 9s

File: docs/migration_guide.md (hunk 1 above)
Comment (no23reason) [reply docs/migration_guide.md]: I believe TypeScript should be spelled like that (capital letters). Also, I think the sentence should read
> If you are using this library from **a** TypeScript project, it is necessary to upgrade to TypeScript of version at least 3.3.4000.

Comment (pbenes) [reply docs/migration_guide.md]: I will fix the TypeScript capital letters, good catch! As for the a/the I would ask Zhenya, what would you think @evgeniabogranskaya ?

Comment (evgeniabogranskaya) [reply docs/migration_guide.md]: @pbenes, @no23reason, I am confused =(
This is the version of this sentence that I see: https://github.com/gooddata/gooddata-ui/pull/227/files#diff-dc59405920762941dbf802c2eeb8001aR10
I agree with the capitalization: TypeScript
Re a/the: if we are talking about any TypeScript project that a user can have, then yes it should be 'a'.
So, the final version is:
`If you are using this library from a TypeScript project, upgrade it to version 3.3.4000 at least.`
Do we all agree?

Comment (pbenes) [reply docs/migration_guide.md]: I'm fine with this (the text you see here above is outdated, the current version is always visible in the `Files changed` tab of this pull-request)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/84/ : SUCCESS in 1m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/65/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/85/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/23/ : SUCCESS in 20s

**Pull request merged**